src/ciadmin/generate/worker_pools.py

```diff
@@ -27,11 +27,27 @@ def is_invalid_aws_instance_type(invalid_instances, zone, instance_type):
 
 
 def is_invalid_gcp_instance_type(invalid_instances, zone, machine_type):
-    family, _ = machine_type.split("-", 1)
-    return any(
-        (zone in entry["zones"]) and (family in entry["families"])
```

Comment:
I don't mind either way, but you could fairly easily keep this more concise code by adding `(suffix in entry.get("suffixes", []))` to it.

Reply:
I ended up doing something different here, since we want it to always match if no suffixes are provided in the config.